src/ngraph/frontend/onnxifi/onnxifi.cpp

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+* Copyright 2017-2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <cstdint>
+#include <onnxifi.h>
+
+extern "C" {
+
+onnxStatus onnxGetBackentIDs(onnxBackendID* backendIDs, std::size_t* numBackends)
```

Comment:
Typo in `onnxGetBackentIDs` -> `onnxGetBackendIDs`

Reply:
Done